lkmpg.tex

```diff
@@ -1016,7 +1016,7 @@ \subsection{Registering A Device}
 int alloc_chrdev_region(dev_t *dev, unsigned baseminor, unsigned count, const char *name);
 \end{code}
 
-The choose of two different functions depend on whether you know the major numbers for your device.
+The choice of two different functions depend on whether you know the major numbers for your device.
```

Comment:
Should we use "depends on" ?

Reply:
Ah, of course. Maybe even "The choice between two different functions depends on ..."?

Reply:
Sounds good! Let's rephrase.